### PR TITLE
Misc docs

### PR DESCRIPTION
**Pull request by regit**
Minor documentation update.

- [x] I have read the contributing guide lines at https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
- [x] I have signed the Open Information Security Foundation contribution agreement at https://suricata-ids.org/about/contribution-agreement/
- [x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Describe changes:
- Fix doc on geoip keyword
- Some display update

[PRScript](https://redmine.openinfosecfoundation.org/projects/suricata/wiki/PRscript) output:
- PR regit: https://buildbot.openinfosecfoundation.org/builders/regit/builds/480
- PR regit-pcap: https://buildbot.openinfosecfoundation.org/builders/regit-pcap/builds/261

### CODE REVIEW

File: doc/userguide/rules/header-keywords.rst [hunk 1]
```diff
@@ -142,19 +142,23 @@ API of MaxMind.
...
+direction you would like to match
 
-  both: both directions have to match with the given geoip (geopip's)
-  any: one of the directions have to match with the given geoip ('s).
-  dest: if the destination matches with the given geoip.
-  src: the source matches with the given geoip.
+====== =============================================================
+Option Description
+====== =============================================================
+both   Both directions have to match with the given geoip (geopip's)
```
Comment (jlucovsky) [inline doc/userguide/rules/header-keywords.rst]: nit: the single apostrophe shouldn't be used for these examples.

File: doc/userguide/rules/header-keywords.rst [hunk 2]
```diff
@@ -142,19 +142,23 @@ API of MaxMind.
...
+any    One of the directions have to match with the given geoip ('s).
```
Comment (jlucovsky) [inline doc/userguide/rules/header-keywords.rst]: nit: s/have/has/

File: doc/userguide/rules/header-keywords.rst [hunk 3]
```diff
@@ -495,43 +499,75 @@ The following lists the meaning of all ICMP types. When a code is not listed,
...
+|           +-----------+------------------------------------------------------------------------+
+|           | 1         |  Redirect Datagram for the Host                                        |
+|           +-----------+------------------------------------------------------------------------+
+|           | 2         |  Redirect Datagram for the Type of Service and Network                 |
+|           +-----------+------------------------------------------------------------------------+
+|           | 3         |  Redirect Datagram for the Type of Service and Host                    |
+|           +-----------+------------------------------------------------------------------------+
+|           | 0         |  Normal router advertisement                                           |
+|           +-----------+------------------------------------------------------------------------+
+|           | 16        |  Doest not route common traffic                                        |
```
Comment (jlucovsky) [inline doc/userguide/rules/header-keywords.rst]: I couldn't find `16` in the rfc ... `Doest` --> should be `Doesn't`?

File: doc/userguide/rules/header-keywords.rst [hunk 4]
```diff
@@ -495,43 +499,75 @@ The following lists the meaning of all ICMP types. When a code is not listed,
...
-            - 1         - Missing a Required Option
-            - 2         - Bad Length
-40          - 0         - Bad SPI
-            - 1         - Authentication Failed
-            - 2         - Decompression Failed
-            - 3         - Decryption Failed
-            - 4         - Need Authentication
-            - 5         - Need Authorization
-==========  ==========  =========================================================================
++-----------+-----------+------------------------------------------------------------------------+
```
Comment (jlucovsky) [inline doc/userguide/rules/header-keywords.rst]: nit: Not sure if rst formatting will render the description differently based on the number of spaces between the `|` and the text that follows it. A couple of the items in the `Description` column have inconsistent white space between the `|` and the text.

File: doc/userguide/rules/header-keywords.rst (hunk 2 above)
Comment (regit) [reply doc/userguide/rules/header-keywords.rst]: Hey, this is not my text ;) Fixing it :)

File: doc/userguide/rules/header-keywords.rst (hunk 1 above)
Comment (regit) [reply doc/userguide/rules/header-keywords.rst]: Fixed.

File: doc/userguide/rules/header-keywords.rst (hunk 4 above)
Comment (regit) [reply doc/userguide/rules/header-keywords.rst]: Don't think so but fixing it anyway.

File: doc/userguide/rules/header-keywords.rst (hunk 3 above)
Comment (regit) [reply doc/userguide/rules/header-keywords.rst]: Good catch on 16, it was supposed to be code 9 !

**Comment by regit** [comment]
Resubmitting with addressed comments.

**Pull request closed without merging**